pom.xml

```diff
@@ -11,7 +11,8 @@
   <version>1.11.2-1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Jenkins jQuery plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
+  <description>This plugin is a library plugin for other plugins to share common jQuery. It also allows users to use jQuery on each view descriptions.</description>
```

Comment:
> if we change the value in pom is reflected in jenkins UI, potentially making confusion on users

Remove it. Jenkins doesn't show it anyway because it's so useless:

https://github.com/jenkinsci/jenkins/blob/33f458d358a03cb4454150e57594a6cc2e7c24b9/core/src/main/java/hudson/model/UpdateSite.java#L999